introduction.md

```diff
@@ -0,0 +1,2 @@
+$ echo 'Hello! I am here to teach people about GitHub.' > introduction.md
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.